.github/workflows/centos.yml

```diff
@@ -0,0 +1,164 @@
+        if: ${{ env.RELEASE_CREATED == 'true' }}
+        id: get_release
+        uses:
+          bruceadams/get-release@v1.2.3
+
+      - name: install requirements
+        run: |
+          yum install -y epel-release
+          yum install -y git redhat-lsb-core make wget centos-release-scl scl-utils rpm-build
+          yum install -y cmake3 devtoolset-9
+
+      - name: Checkout OR-Tools
+        run: |
+          git clone $GITHUB_SERVER_URL/$GITHUB_REPOSITORY.git -b ${BRANCH_NAME} .
+
+      - name: Checkout xpressmp linux
+        run: |
+          git clone https://${{ secrets.ACCESS_TOKEN }}@github.com/rte-france/xpress-mp.git ${{ env.XPRESS_INSTALL_DIR }}
+
+      - name: set Xpress variables
+        run: |
+          cd ${{ env.XPRESS_INSTALL_DIR }}
+          echo "XPRESSDIR=$PWD" >> $GITHUB_ENV
+          echo "XPRESS=$PWD/bin" >> $GITHUB_ENV
+          echo "XPRESS_LIB=$PWD/lib" >> $GITHUB_ENV
+
+      - name: Download sirius
+        run: |
+          wget https://github.com/rte-france/sirius-solver/releases/download/${{ env.SIRIUS_RELEASE_TAG }}/centos-7_sirius-solver.zip
+          unzip *.zip
+          mv centos-7_sirius-solver-install ${{ env.SIRIUS_INSTALL_DIR }}
+
+      - name: set Sirius variables
+        run: |
+          cd ${{ env.SIRIUS_INSTALL_DIR }}
+          echo "SIRIUS=$PWD/lib" >> $GITHUB_ENV
+          echo "SIRIUS_CMAKE_DIR=$PWD/cmake" >> $GITHUB_ENV
+
+      - name: Configure OR-Tools
+        run: |
+          source /opt/rh/devtoolset-9/enable
+          cmake3 -S . -B build -DBUILD_SAMPLES=OFF -DBUILD_EXAMPLES=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="build/install" -DBUILD_SHARED_LIBS=${{ matrix.shared }} -DBUILD_DEPS=ON -DUSE_SIRIUS=${{ matrix.sirius }} -Dsirius_solver_DIR="${{ env.SIRIUS_CMAKE_DIR }}" -DUSE_XPRESS=${{ matrix.xprs }} -DXPRESS_ROOT="${{ env.XPRESSDIR }}" -DBUILD_PYTHON=${{ matrix.python }} -DBUILD_JAVA=${{ matrix.java }} -DBUILD_DOTNET=${{ matrix.dotnet }}
+
+      - name: Build OR-Tools Linux
+        run: |
+          source /opt/rh/devtoolset-9/enable
+          cmake3 --build build --config Release --target all install -j4
+
+      - name: Prepare OR-Tools install
+        if: ${{ matrix.publish-or == 'ON' }}
+        id: or-install
+        run: |
+          find build/install/bin -type f ! \( -name 'protoc*' -o -name 'scip' -o -name 'fz' \) -delete
+          cd build
+          ARCHIVE_NAME="ortools_cxx${{ steps.names.outputs.appendix_with_shared }}.zip"
+          ARCHIVE_PATH="$PWD/${ARCHIVE_NAME}"
+          zip -r $ARCHIVE_PATH ./install
+          echo "::set-output name=archive_name::$ARCHIVE_NAME"
+          echo "::set-output name=archive_path::$ARCHIVE_PATH"
+
+      - name: Upload OR-Tools install artifact
+        uses: actions/upload-artifact@v2
+        if: ${{ matrix.publish-or == 'ON' }}
+        with:
+          name: ${{ steps.or-install.outputs.archive_name }}
+          path: ${{ steps.or-install.outputs.archive_path }}
```

Comment:
I'm not sure why we should upload this artifact, since it appears to be unused. Remove this step ? Possibly publish only when `env.RELEASE_CREATED == 'false'` ?

Reply:
it could be used to test the CI without a release.
Yes we could upload it if no release is created